specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -2253,6 +2381,13 @@
       "required": true,
       "type": "string"
     },
+    "billingAccountIdParameter": {
+      "name": "billingAccountId",
+      "in": "path",
+      "description": "Azure Billing Account ID.",
+      "required": true,
+      "type": "string"
```

Comment:
This causes a breaking change in the sdk. Refer to this [comment](https://github.com/Azure/azure-sdk-for-node/pull/2536/files#r176623796).
This parameter should have `"x-ms-parameter-location": "method"` extension applied.

Whoever merged the PR last time did not catch that [grainParameter](https://github.com/Azure/azure-rest-api-specs/pull/2742/files#diff-dca9b11ab859f276378213344e091b34R2431) defined below also has missing`"x-ms-parameter-location": "method"` extension

Reply:
This should have been caught when this [PR](https://github.com/Azure/azure-rest-api-specs/pull/2625/files#diff-dca9b11ab859f276378213344e091b34R2103) was sent 9 days ago.

Reply:
@marstr - Please take a look.

Reply:
Added "x-ms-parameter-location": "method" for grainParameter.
billing accounts parameter requires some other changes from our end. Will update that shortly.

Reply:
Sure. But please make sure to add `"x-ms-parameter-location": "method"` for `"billingAccountIdParameter"` as well.